python/ray/air/tests/test_remote_storage.py

```diff
@@ -131,6 +131,46 @@ def test_upload_exclude_multimatch(temp_data_dirs):
     assert_file(False, tmp_target, "subdir_exclude/something/somewhere.txt")
 
 
+@pytest.mark.parametrize("no_fsspec", [False, True])
+def test_upload_local_exclude_multi(temp_data_dirs, no_fsspec):
+    if no_fsspec:
+        with patch("ray.air._internal.remote_storage.fsspec", None):
+            return test_upload_local_exclude_multi(temp_data_dirs, no_fsspec=False)
+
+    tmp_source, tmp_target = temp_data_dirs
+
+    upload_to_uri(tmp_source, tmp_target, exclude=["*_exclude.txt", "*_exclude/*"])
```

Comment:
The tests were copied from existing tests that test the same behavior with an in-memory filesystem. The main difference is that in the second case, multiple patterns are tested (not just one).

The `exclude/` pattern won't work as it will not match the files contained within the directory - it always has to be `exclude/*`